base/stream.jl

```diff
@@ -1100,7 +1100,7 @@ function uv_write(s::LibuvStream, p::Ptr{UInt8}, n::UInt)
     if status < 0
         throw(_UVError("write", status))
     end
-    return Int(n)
+    return n % Int64
 end
```

Comment:
Doesn't this change the return type on 32-bit systems?

Reply:
Yes, like I said I thought ok, intentional, but if not I can fix that. Do you really rely on in Int32?